File: AssayDashboard/Source/Hologic.AssayDashboard.AssayParser/Models/ReagentTypeDescription.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Xml.Serialization;

namespace Hologic.AssayDashboard.AssayParser
{
    [XmlRoot("ReagentTypeDescriptions")]
    public class ReagentTypeDescription
    {
        [XmlElement("CalibratorTypeDescription")]
        public List<Calibrator> calibrators;

        [XmlElement("ControlTypeDescription")]
        public List<Control> controls;

        [XmlElement("AssayReagentTypeDescription")]
        public List<Reagent> reagents;
    }
}

File: AssayDashboard/Source/Hologic.AssayDashboard.Client/src/app/assayviewrepo/assayviewrepo-models/reagentTypeDescription.ts

/*
// ----------------------------------------------------------------
// Copyright (c) 2017 Hologic, Inc.  All Rights Reserved.
// ----------------------------------------------------------------
*/
import { Control } from "app/assayviewrepo/assayviewrepo-models/control";
import { Calibrator } from "app/assayviewrepo/assayviewrepo-models/calibrator";
import { Reagent } from "app/assayviewrepo/assayviewrepo-models/reagent";

export class ReagentTypeDescription {
    controls: Control[];
    calibrators: Calibrator[];
    reagents: Reagent[];

   /* constructor() {
        this.controls = [];
        this.calibrators = [];
        this.reagents = [];
    }*/
}